modules/planning/scenarios/park_and_go/stage_adjust.cc

```diff
@@ -48,10 +48,22 @@ Stage::StageStatus ParkAndGoStageAdjust::Process(
     AERROR << "ParkAndGoStageAdjust planning error";
     return StageStatus::ERROR;
   }
-  const bool ready_to_cruise =
+  bool is_ready_to_cruise =
```

Comment:
Keep `const`?

Reply:
I thought you said "Do not use const bool" before.

Reply:
Fixed. Thank you!